databaseutil/databaseutil.go

```diff
@@ -46,6 +49,12 @@ func InsertIntoUsersTable(
 			return nil
 		}
 
+		if pgErr, ok := err.(*pq.Error); ok {
+			if pgErr.Code == "23505" {
```

Comment:
Consider making this a const int at the top of the file.

Reply:
it's a string. See above comment about plans for this code. I was planning on waiting for another use case before generalizing though